### PR TITLE
Hania new component structure

### PR DESCRIPTION
**Pull request by HaniaNassalska**

### CODE REVIEW

File: src/common/Navigation/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const Navigation = () => {
 
       </NavigationBoxStyled>
       <Routes>
-        <Route path="/movies" element={<Movies />} />
```
Comment (KubaTuta) [inline src/common/Navigation/index.js:48]: here you can path="/" co it would be main site

File: src/common/Navigation/index.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const Navigation = () => {
...
+        <Route path="/movies" element={<Movies/>} />
```
Comment (KubaTuta) [inline src/common/Navigation/index.js]: here you can path="/" co it would be main site

Comment (HaniaNassalska) [reply src/common/Navigation/index.js]: oki

**Comment by KubaTuta** [comment]
Great job like new look

**Review by KubaTuta — APPROVED** [review]
Good job like new look

**Pull request merged**